src/Microsoft.AspNetCore.SignalR.Core/HubEndPoint.cs

```diff
@@ -450,30 +481,34 @@ private bool IsChannel(Type type, out Type payloadType)
             {
                 await SendMessageAsync(connection, CompletionMessage.WithError(invocationId, ex.Message));
             }
+            finally
+            {
+                if (connection.ActiveRequestCancellationSources.TryRemove(invocationId, out var cts))
```

Comment:
Ah wait we do this!

Reply:
?

Reply:
So we'll get warnings if the steam on the server ends and then the client sends a cancellation?

Reply:
I made it a debug log https://github.com/aspnet/SignalR/pull/979/files#diff-3f9cc189cbcb6b225bb8d6cb6ecdeb21R282